src/main/java/io/github/opencubicchunks/cubicchunks/core/server/CubicAnvilChunkLoader.java

```diff
@@ -100,7 +100,7 @@ private ICubeIO getCubeIO() {
     }
 
     @Override public boolean writeNextIO() {
-        return getCubeIO().writeNextIO();
+        throw new UnsupportedOperationException();
```

Comment:
Why?

Reply:
because it's only called by `AnvilChunkLoader#flush()`, which is overridden by `CubicAnvilChunkLoader#flush()`, and the overridden implementation doesn't call `writeNextIO()`. additionally, it seems very gross to expose the asynchronous implementation details of `ICubeIO` to the world at large.

Reply:
I've seen mods call methods from AnvilChunkLoader before, which is why I tried to implememt as many of them as possible.

Reply:
alright, i put it back to how it was before.